Sources/Purchasing/NonSubscriptionTransaction.swift

```diff
@@ -23,22 +23,38 @@ public final class NonSubscriptionTransaction: NSObject {
     /// The date that App Store charged the user’s account.
     @objc public let purchaseDate: Date
 
-    /// The unique identifier for the transaction.
+    /// The unique identifier for the transaction created by RevenueCat.
     @objc public let transactionIdentifier: String
 
+    /// The unique identifier for the transaction created by the Store.
+    @objc public let storeTransactionIdentifier: String
```

Comment:
I'd make it internal for now, because: 
a) we don't currently expose it for subscriptions, so maybe it's better to avoid confusion
b) it's also not exposed on android, so maybe it's better for parity?
c) we're doing related work with external purchases API, we should ensure that we unify naming before exposing

Reply:
👍🏻 

Will do.